breadbox/breadbox/service/slice.py

```diff
@@ -17,6 +20,63 @@
 from depmap_compute.slice import SliceQuery
 
 
+@dataclass
+class ResolvedSlice:
+    dataset: Dataset
+    label: str
+    given_id: str
+
+
+def resolve_slice_to_components(
```

Comment:
This function sounds like it will be really helpful, but the naming made me a bit confused when I was first reading through. I tend to think of a "slice" being the full set of data belonging to the slice query, whereas I would call the label/given id "slice identifiers".  Would it be possible to rename this to something like `ResolvedSliceIdentifiers` or `DimensionIdentifiers` instead of `ResolvedSlice`?

Reply:
Sure, happy to rename this to `ResolvedSliceIdentifiers`